### PR TITLE
write None value for nullable field

### PR DESCRIPTION
**Pull request by nobo728x**
ValueError is raised for nullable field

### CODE REVIEW

**Comment by tebeka** [comment]
Thanks for improving fastavro!
I'm on the road currently, will look at it end next week.

File: fastavro/writer.py [hunk 1]
```diff
@@ -239,9 +238,15 @@ def write_record(fo, datum, schema):
     their schema."""
     for field in schema['fields']:
         name = field['name']
+        f_type = field['type']
         if name not in datum and 'default' not in field:
-            raise ValueError('no value and no default for %s' % name)
-        write_data(fo, datum.get(name, field.get('default')), field['type'])
+            if 'null' in f_type:
```
Comment (scottbelden) [inline fastavro/writer.py]: I think you can accomplish what you are looking for by just changing the original 
```python
if name not in datum and 'default' not in field:
```

to

```python
if name not in datum and 'default' not in field and 'null' not in field['type']:
```

Comment (nobo728x) [inline fastavro/writer.py]: You are right :+1:

Comment (nobo728x) [inline fastavro/writer.py]: Change done.

**Pull request merged**